shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -473,6 +477,8 @@
 
           // Insert placeholders to fill remaining space
           var remaining = this.layout.cols - x;
+          if(isRTL)
+            x = (windowWidth - this.layout.gridItemWidth) - x;
```

Comment:
Space after if and braces around following statement, to follow style with the rest of the file.